src/main/java/de/blazemcworld/blazinggames/crates/DeathCrateKey.java

```diff
@@ -89,5 +98,28 @@ public static String getKeyULID(ItemStack item) {
     }
 
     public record DeathCrateKeyContext(String crateId) implements ItemContext {
+        public static DeathCrateKeyContext parse(Player player, String string) throws ParseException {
+            if (!string.contains(":")) {
+                string = "ulid:" + string;
+            }
+
+            String[] split = string.split(":", 2);
+
+            switch (split[0]) {
```

Comment:
suggestion: add toLowerCase for user friendliness
```suggestion
            switch (split[0].toLowerCase()) {
```